CMakeSettings.json

```diff
@@ -6,7 +6,7 @@
   // Open Enclave SDK from Visual 2017. It is based on the default
```

Comment:
why is this rendered with red?

Reply:
I don't know neither.. maybe it's a json file?